teuthology/orchestra/run.py

```diff
@@ -178,6 +182,14 @@ def _raise_for_status(self):
                 # signal; sadly SSH does not tell us which signal
                 raise CommandCrashedError(command=self.command)
             if self.returncode != 0:
+                if self.scan_tests_errors:
+                    test, error = ErrorScanner().scan(test_names=self.scan_tests_errors)
```

Comment:
It might even be a good idea to wrap this statement in a `try: / except Exception` clause for fault-tolerance.